### PR TITLE
Feature/load default selectasync

### LINKED ISSUE #743: feature: auto select default tax
In InvoiceForm, OfferForm and QuoteForm the tax input selector should automatically set the default tax when it loads.
How can it be implemented?

### PR DESCRIPTION
**Pull request by johnvan7**
## Description
Resolve #743 

## Checklist

- [x] I have tested these changes
- [ ] I have updated the relevant documentation
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the codebase
- [x] My changes generate no new warnings or errors
- [x] The title of my pull request is clear and descriptive

### CODE REVIEW

File: frontend/src/components/SelectAsync/index.jsx [hunk 1]
```diff
@@ -11,6 +11,8 @@ export default function SelectAsync({
   outputValue = '_id',
   value,
   onChange,
+  loadDefault = true,
```
Comment (salahlalami) [inline frontend/src/components/SelectAsync/index.jsx:14]: this should be loadDefault = false, unless the user make it true

File: frontend/src/modules/InvoiceModule/Forms/InvoiceForm.jsx [hunk 2]
```diff
@@ -239,7 +239,6 @@ function LoadInvoiceForm({ subTotal = 0, current = null }) {
               ]}
             >
               <SelectAsync
-                  value={taxRate}
```
Comment (salahlalami) [inline frontend/src/modules/InvoiceModule/Forms/InvoiceForm.jsx:242]: i think we need to keep this for update

**Review by salahlalami — CHANGES_REQUESTED** [review]

**Comment by johnvan7** [comment]
Done

**Comment by johnvan7** [comment]
Also added `loadDefault={true}` in InvoiceForm, OfferForm, QuoteForm and PaymentForm because tax and paymentMode have the "isDefault" prop.

**Pull request merged**